frameworks/autosklearn/setup.sh

```diff
@@ -19,12 +19,15 @@ PIP install --no-cache-dir -r $HERE/requirements.txt
 if [[ "$VERSION" == "stable" ]]; then
     PIP install --no-cache-dir -U ${PKG}
 elif [[ "$VERSION" =~ ^[0-9] ]]; then
+    # Provided a version number
     PIP install --no-cache-dir -U ${PKG}==${VERSION}
 else
-#    PIP install --no-cache-dir -e git+${REPO}@${VERSION}#egg=${PKG}
     TARGET_DIR="${HERE}/lib/${PKG}"
     rm -Rf ${TARGET_DIR}
-    git clone --depth 1 --single-branch --branch ${VERSION} --recurse-submodules ${REPO} ${TARGET_DIR}
+    git clone  --recurse-submodules ${REPO} ${TARGET_DIR}
+    cd ${TARGET_DIR}
+    git checkout $VERSION
+    cd ${HERE}
```

Comment:
we don't need to pull the entire repo just to run `autosklearn`, why removing the `--depth`, `--single-branch` and `--branch` arguments?